apps/televisit-demo/backend/signinhook/index.js

```diff
@@ -0,0 +1,38 @@
+const AWS = require('aws-sdk');
```

Comment:
Please ignore this comment. This file is from the following template in the [chat demo](https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/chat):

https://github.com/aws-samples/amazon-chime-sdk/blob/a0527f499c9a5336a3c5351ce9aec295e05af115/apps/chat/src/backend/serverless/template.yaml#L249